system-tests/lib/crecli/settings.go

```diff
@@ -125,3 +168,34 @@ func PrepareCRECLISettingsFile(workflowOwner, capRegAddr, workflowRegistryAddr c
 
 	return settingsFile, nil
 }
+
+func PrepareCRECLIWorkflowSettingsFile(profile string, workflowOwner common.Address, workflowName string) (*os.File, error) {
+	settingsFile, err := os.CreateTemp("", CRECLIWorkflowSettingsFile)
```

Comment:
let's not create it at all if we won't use it?

Reply:
we are using it.